application/ingestr/kafka-cmd.sh

```
@@ -0,0 +1,199 @@
+#/usr/bin/env bash
```

Shellcheck (0.10.0)

[error] Use #!, not just #, for the shebang.

(SC1113)